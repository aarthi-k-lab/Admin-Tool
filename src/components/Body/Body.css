.body-container {
  display: flex;
  flex-flow: nowrap;
  flex-direction: column;
  flex-grow: 1;
  width: 100%;
}

.body {
  display: flex;
  flex-flow: nowrap;
  flex-direction: row;
  flex-grow: 1;
}
