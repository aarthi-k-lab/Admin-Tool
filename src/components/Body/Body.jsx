import React from 'react';
import PropTypes from 'prop-types';

import './Body.css';

const Body = ({ children }) => (
  <section styleName="body">
    { children }
  </section>
);

Body.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Body;
