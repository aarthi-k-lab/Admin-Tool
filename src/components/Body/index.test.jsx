import React from 'react';
import renderer from 'react-test-renderer';
import Body from '.';

describe('<Body />', () => {
  test('snapshot test', () => {
    const tree = renderer
      .create(<Body>content</Body>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
