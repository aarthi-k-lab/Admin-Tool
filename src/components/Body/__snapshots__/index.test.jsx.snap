// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Body /> snapshot test 1`] = `
<div
  className="components-Body-Body__body-container--3Iujq"
>
  <section
    className="components-Body-Body__body--3PVBY"
  >
    content
  </section>
</div>
`;
